pkg/apis/pipeline/v1beta1/pipeline_validation.go

```diff
@@ -269,11 +270,42 @@ func validatePipelineResults(results []PipelineResult) (errs *apis.FieldError) {
 				"value").ViaFieldIndex("results", idx))
 		}
 
+		if !taskContainsResult(result.Value.StringVal, pipelineTaskNames) {
+			return errs.Also(apis.ErrInvalidValue("referencing a nonexistent task",
```

Comment:
It might be better to use `errs = errs.Also` instead of a return statement here. The reason is that if multiple references exist to non-existent tasks, this function will only return an error message about one of them, forcing the person to fix the errors one by one.

Reply:
done, thank you for the feedback.